docs/guides/config.json_directives.md

```diff
@@ -0,0 +1,143 @@
+# Getting the bot started
+
+## config.json directives
+
+config.json, a file that can store data, that is used by Concord on runtime. You will probably wish to set it up, so, here is a guide explaining what are the purposes of everything in there.
+
+```json
+{
+  "logging": {
+    "level": "trace",
+    "filename": "bot.log",
+    "quiet": false,
+    "overwrite": true,
+    "use_color": true,
+    "http": {
+      "enable": true,
+      "filename": "http.log"
+    },
+    "disable_modules": ["WEBSOCKETS", "USER_AGENT"]
+  },
+  "discord": {
+    "token": "YOUR-BOT-TOKEN",
+    "default_prefix": {
+      "enable": false,
+      "prefix": "YOUR-COMMANDS-PREFIX"
+    }
+  }
+}
+```
+
+### logging
+
+#### logging.level
```

Comment:
As suggested by @tarbomb, does this look better as a table? You can make a separate one for `logging` and `discord`, and let us know if it looks any better.